wscript

```diff
@@ -39,7 +39,7 @@ def options(opt):
 def configure(conf):
   env = conf.env
 
-  env.append_unique('CXXFLAGS', ['-O2', '-Wall', '-g', '-pipe', '-fno-omit-frame-pointer', '-pthread'])
+  env.append_unique('CXXFLAGS', ['-O2', '-Wall', '-g', '-pipe', '-fno-omit-frame-pointer', '-pthread', '-msse4.2'])
```

Comment:
I think it is better to remove this, as this creates platform-dependent binaries.
It will become a problem when building binary packages for example.